### PR TITLE
should fail if recipient address is invalid

### LINKED ISSUE #126: [TASK] Add additional negative tests for EVM Anchor
# Overview

**Test Cases**
- As a relayer I should reject a relayer address that is not my beneficiary 
- As a relayer I should reject an invalid proof
- As a relayer I should reject a fee that is less than requested

### PR DESCRIPTION
**Pull request by dharjeezy**
closes #126

### CODE REVIEW

File: tests/test/evmTransactionRelayer.test.ts [hunk 1]
```diff
@@ -24,6 +24,9 @@ import temp from 'temp';
 import { LocalChain } from '../lib/localTestnet.js';
 import { calcualteRelayerFees, WebbRelayer } from '../lib/webbRelayer.js';
 import getPort, { portNumbers } from 'get-port';
+import {IAnchor} from "@webb-tools/interfaces";
```
Comment (drewstone) [inline tests/test/evmTransactionRelayer.test.ts:27]: Add spaces between brackets and please use single quotes. Maintain consistency.

**Review by drewstone — CHANGES_REQUESTED** [review]
Looks good! Just fix up some stylistic issues and this is good to go.

**Review by shekohex — CHANGES_REQUESTED** [review]
Looks good to me, just a few changes to the formatting.
Please run:
```sh
./node_modules/.bin/prettier -w '**/*.ts'
```

Or better add a script In the package.json like so

```json
"format": "prettier -w **/*.ts",
```
and then you can run `yarn format`.

**Review by shekohex — APPROVED** [review]
LGTM!

**Pull request merged**